infra/profile.py

```diff
@@ -34,16 +56,21 @@ def bench(profile):
     os.mkdir(profile_dir)
   except FileExistsError:
     print(f'{profile_dir} exists, overwriting contents')
-    
+
   for mode in modes:
-    (name, runmode, options) = mode
-    subprocess.call(f'cargo run --release {profile} --run-mode {runmode} {options} -o {profile_dir}/{name}', shell=True)
+    subprocess.call(f'cargo run --release {profile} {get_eggcc_options(mode, profile_dir)} -o {profile_dir}/{mode}', shell=True)
 
-    with open(f'{profile_dir}/{name}-args') as f:
+    with open(f'{profile_dir}/{mode}-args') as f:
       args = f.read().rstrip()
     
     # TODO for final nightly results, remove `--max-runs 2` and let hyperfine find stable results
-    subprocess.call(f'hyperfine --warmup 1 --max-runs 2 --export-json {profile_dir}/{name}.json "{profile_dir}/{name} {args}"', shell=True)
+    subprocess.call(f'hyperfine --warmup 1 --max-runs 2 --export-json {profile_dir}/{mode}.json "{profile_dir}/{mode} {args}"', shell=True)
+
+def get_llvm(runMethod, benchmark):
+  files = [f for f in glob(f'./tmp/bench/{benchmark}/llvm-{runMethod}/*') if "init" not in f]
```

Comment:
I think this is the kind of code that could get us in trouble, especially if we change the naming conventions.

Could you please instead change the code that emits llvm to use the same naming convention as above? e.g. `egglog_noopt_bril_llvm_opt.ll`